.kokoro/samples-test.sh

```diff
@@ -20,7 +20,7 @@ export NPM_CONFIG_PREFIX=${HOME}/.npm-global
 
 # Setup service account credentials.
 export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
-export GCLOUD_PROJECT=long-door-651
+export GCLOUD_PROJECT=ucaip-sample-tests
```

Comment:
I'd rather not change `.kokoro/samples-test.sh` which is auto-generated.

You can just set the `project` to `ucaip-sample-tests` in the tests, and ensure that this is passed by `exec` to the samples you're executing.

Reply:
Just to be perfectly explicit, you want me to hardcode the project id into every test, yeah?